<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPassword</class>
 <widget class="QDialog" name="ResetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="newPasswordLayout">
      <item>
       <widget class="QLabel" name="newPassword">
        <property name="text">
         <string>New Password:   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="newPasswordInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="renterPasswordLayout">
      <item>
       <widget class="QLabel" name="renterPasswordLabel">
        <property name="text">
         <string>Renter Password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="renterPasswordInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttons">
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
